contracts/pools/SyndicatePoolBase.sol

```diff
@@ -320,7 +320,7 @@ abstract contract SyndicatePoolBase is IPool, SyndicateAware, ReentrancyGuard {
   ) external {
     // sync and call processRewards
     _sync();
-    _processRewards(msg.sender, useSSYN, false);
+    _processRewards(msg.sender, useSSYN, true);
```

Comment:
Are you sure this makes sense? My impression was that that false is wanted. I am not sure, but I would be very careful about changing so much the logic. That is what Illuvium has online right now. 
Anyway, is it possible that that is what is causing the discrepancy you were talking about?

Reply:
What do you think, can we just remove that _sync() from there and be fine with it?

Reply:
done